### PR TITLE
changes urls in Chinese translation of docs.

### PR DESCRIPTION
**Pull request by jingsam**
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by dingjs — DISMISSED** [review]
LGTM

**Comment by kannappanr** [comment]
@jingsam Can you please resolve the conflict in this PR?

File: README_zh_CN.md [hunk 1]
```diff
@@ -87,19 +87,19 @@ go get -u github.com/minio/minio
...
-- [`mc`快速入门](https://docs.minio.io/docs/minio-client-quickstart-guide)
-- [使用 `aws-cli`](https://docs.minio.io/docs/aws-cli-with-minio)
-- [使用 `s3cmd`](https://docs.minio.io/docs/s3cmd-with-minio)
-- [使用 `minio-go` SDK](https://docs.minio.io/docs/golang-client-quickstart-guide)
+- [Minio纠删码入门](https://docs.minio.io/cn/minio-erasure-code-quickstart-guide)
+- [`mc`快速入门](https://docs.minio.io/cn/minio-client-quickstart-guide)
+- [使用 `aws-cli`](https://docs.minio.io/cn/aws-cli-with-minio)
+- [使用 `s3cmd`](https://docs.minio.io/cn/s3cmd-with-minio)
+- [使用 `minio-go` SDK](https://docs.minio.io/cn/golang-client-quickstart-guide)
 - [Minio文档](https://docs.minio.io)
```
Comment (kannappanr) [inline README_zh_CN.md]: Should `https://docs.minio.io` be changed to `https://docs.minio.io/cn/` ?

File: docs/zh_CN/large-bucket/README.md [hunk 2]
```diff
@@ -37,11 +37,11 @@ minio server http://host{1...4}/export{1...16}
...
-- [使用 aws-cli](https://docs.minio.io/docs/aws-cli-with-minio)
-- [使用 s3cmd](https://docs.minio.io/docs/s3cmd-with-minio)
-- [使用 minio-go SDK](https://docs.minio.io/docs/golang-client-quickstart-guide)
-- [Minio文档](https://docs.minio.io)
\ No newline at end of file
+- [mc快速入门](https://docs.minio.io/cn/minio-client-quickstart-guide)
+- [使用 aws-cli](https://docs.minio.io/cn/aws-cli-with-minio)
+- [使用 s3cmd](https://docs.minio.io/cn/s3cmd-with-minio)
+- [使用 minio-go SDK](https://docs.minio.io/cn/golang-client-quickstart-guide)
+- [Minio文档](https://docs.minio.io)
```
Comment (kannappanr) [inline docs/zh_CN/large-bucket/README.md]: Should https://docs.minio.io be changed to https://docs.minio.io/cn/ ?

**Comment by kannappanr** [comment]
@dingjs Can you please review again?

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

File: docs/zh_CN/large-bucket/README.md (hunk 2 above)
Comment (dingjs) [reply docs/zh_CN/large-bucket/README.md]: Yes,You are right about it.

**Review by dingjs — CHANGES_REQUESTED** [review]
Hi, jingsam 
https://docs.minio.io should be changed to https://docs.minio.io/cn/

**Comment by jingsam** [comment]
I don't get it. Do you mean https://docs.minio.io/docs should change to https://docs.minio.io/cn/docs?

**Comment by kannappanr** [comment]
> I don't get it. Do you mean https://docs.minio.io/docs should change to https://docs.minio.io/cn/docs?

No. https://docs.minio.io should be changed to https://docs.minio.io/cn/

**Comment by deekoder** [comment]
@jingsam can you make the links to have docs.minio.io/cn?

File: docs/zh_CN/large-bucket/README.md (hunk 2 above)
Comment (deekoder) [reply docs/zh_CN/large-bucket/README.md]: ping @jingsam  can the links be updated.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by harshavardhana** [comment]
Closing this PR as stale @jingsam - please re-open with conflicts fixed and addressing all the comments.

**Pull request closed without merging**